engines/web/app/views/web/angular_templates/cookies_policy.html.haml

```diff
@@ -44,32 +44,20 @@
 %p
   = t 'legal.cookies_policy.essential_cookies_note'
 
-- if Spree::Config.cookies_policy_matomo_section || Spree::Config.cookies_policy_ga_section
+- if Spree::Config.cookies_policy_matomo_section
   %h2
     = t 'legal.cookies_policy.statistics_cookies'
   %p
     = t 'legal.cookies_policy.statistics_cookies_desc'
-
-  - if Spree::Config.cookies_policy_ga_section
-    %p
-      = t 'legal.cookies_policy.statistics_cookies_analytics_desc_html'
-  - if Spree::Config.cookies_policy_matomo_section
-    %p
-      = t 'legal.cookies_policy.statistics_cookies_matomo_desc_html'
+  %p
+    = t 'legal.cookies_policy.statistics_cookies_matomo_desc_html'
 
   %table{ng: { controller:"CookiesPolicyModalCtrl"}}
-    - if Spree::Config.cookies_policy_ga_section
-      = render_cookie_entry( "__utma", t( "legal.cookies_policy.cookie_analytics_utma_desc" ) )
-      = render_cookie_entry( "__utmt", t( "legal.cookies_policy.cookie_analytics_utmt_desc" ) )
-      = render_cookie_entry( "__utmb", t( "legal.cookies_policy.cookie_analytics_utmb_desc" ) )
-      = render_cookie_entry( "__utmc", t( "legal.cookies_policy.cookie_analytics_utmc_desc" ) )
-      = render_cookie_entry( "__utmz", t( "legal.cookies_policy.cookie_analytics_utmz_desc" ) )
-    - if Spree::Config.cookies_policy_matomo_section
```

Comment:
What if an instance isn't using Matomo. Why would we still display the cookie entry?

Reply:
there a if statement above already.
it used to be  if Spree::Config.cookies_policy_matomo_section || Spree::Config.cookies_policy_ga_section
and now it's just  if Spree::Config.cookies_policy_matomo_section
so this one is not needed anymore.

before
```
if matomo or ga
      HEADER
       if matomo    -- this is the IF you are asking about
            MATOMO
       if ga
            GA
```

after:
```
if matomo
      HEADER
      MATOMO

```